src/manifolds/Euclidean.jl

```diff
@@ -40,6 +40,16 @@ This metric is the default metric for example for the [`Euclidean`](@ref) manifo
 """
 struct EuclideanMetric <: RiemannianMetric end
 
+function basis(M::Euclidean{<:Tuple, ℝ}, x, B::ArbitraryOrthonormalBasis)
+    vecs = [_euclidean_basis_vector(x, i) for i in eachindex(x)]
+    return PrecomputedOrthonormalBasis(vecs)
+end
+
+function basis(M::Euclidean{<:Tuple, ℂ}, x, B::ArbitraryOrthonormalBasis)
```

Comment:
I don't think this corresponds to the basis used above. That basis should be
`[im*x]`.

Reply:
Sorry, wrong file. I meant this as the ONB for the `Circle` case.